jenkins-scripts/ign_common-default-devel-windows-amd64.bat

```diff
@@ -4,13 +4,13 @@ set VCS_DIRECTORY=gz-common
 set PLATFORM_TO_BUILD=x86_amd64
 set IGN_CLEAN_WORKSPACE=true
 
-set DEPEN_PKGS=assimp ffmpeg freeimage gts tinyxml2

```

Comment:
Was this removal an error or there is a reason to remove it in this PR?

Reply:
Humm, this might've happened when I rebased, I think it's an error, I'll revert the change

Reply:
https://github.com/gazebo-tooling/release-tools/pull/753/commits/d513c8d8cbdd57af2be151f14154d6233dddd16d

Reply:
good catch! [6138403](https://github.com/gazebo-tooling/release-tools/pull/753/commits/61384030c3e9a567e801e64b6be92b47038e9095)